src/components/ProductDetail/ProductDetail.js

```diff
@@ -69,6 +71,7 @@ class ProductDetail extends Component {
               vendor={product.vendor}
             />
             <VariantList product={product} variants={product.variants}/>
+            <ProductDetailAddToCart>Add To Cart</ProductDetailAddToCart>
```

Comment:
I don't see `children` used in the `ProductDetailAddToCart` component, is it seems like you could just do `< ProductDetailAddToCart />` here?